javatests/arcs/android/integration/policy/policies.arcs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Google LLC.
```

Comment:
Should this be split into multiple arcs files?
- schemas
- recipes
- policies

Reply:
I like the idea of splitting this up into digestable parts. How about this for the breakdown: 

* particles.arcs  -- contains both schemas and particles
* scenarios.arcs -- contains recipes and policies. 

The latter arcs file seems right to me b/c I plan to have one policy per recipe for each scenario.